apihub-portal-publish/README.md

```diff
@@ -0,0 +1,69 @@
+# API Hub Portal Publishing Sample
+
+![Architecture](./docs/architecture.jpg)
+
+This sample shows how unmanaged APIs can be registered in API Hub, on-ramped as a managed API in Apigee with documentation in an [Apigee developer portal](https://cloud.google.com/apigee/docs/api-platform/publish/portal/build-integrated-portal), synced back to API Hub as a managed deployment, and finally offered through Apigee to developers and agents as a secure, managed API.
+
+## Prerequisites
+- [gcloud CLI](https://cloud.google.com/sdk/docs/install) will be used for automating GCP tasks, see the docs site for installation instructions.
+- [apigeecli](https://github.com/apigee/apigeecli) will be used for Apigee automation, see the docs site for installation instructions.
+- [Apigee](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro) and [Apigee API Hub](https://cloud.google.com/apigee/docs/apihub/what-is-api-hub) provisioned in a GCP region.
+- GCP roles needed:
+  - roles/apigee.apiAdminV2 - needed to deploy an Apigee proxy.
+  - roles/apigee.portalAdmin - needed to manage the Apigee integrated developer portal.
+  - roles/apihub.editor - needed to manage API Hub data
+- An [Apigee Integrated Developer Portal](https://cloud.google.com/apigee/docs/api-platform/publish/portal/build-integrated-portal) needs to be provisioned and visible at in the [Apigee Portals Console](https://console.cloud.google.com/apigee/portals)
+
+## (QuickStart) Setup using CloudShell
+
+Use the following GCP CloudShell tutorial, and follow the instructions in Cloud Shell. Alternatively, follow the instructions below.
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/apigee-samples&cloudshell_git_branch=main&cloudshell_workspace=.&cloudshell_tutorial=apihub-portal-publish/docs/cloudshell-tutorial.md)
+
+## Setup instructions
+
+### Step 1: Set your GCP project environment variables
+
+To begin, set your environment variables to be used in the `env.sh` file.
+
+* `PROJECT_ID` the project where your Apigee organization is located.
+* `REGION` the externally reachable hostname of the Apigee environment group that contains APIGEE_ENV.
```

Comment:
should we include APIHUB_REGION as well as the Apigee runtime and the hub instance region could be different?

Reply:
Pls keep in mind that this change will introduce changes to the script and all related files as well

Reply:
it is but its better to just include this as a separate variable just in case its a different region
API hub is not auto provisioned for trial orgs

Reply:
ok added variable APIHUB_REGION